engine/runtime/src/main/scala/org/enso/compiler/core/IR.scala

```diff
@@ -6474,9 +6477,9 @@ object IR {
 
       /** @inheritdoc */
       override def toString: String =
+        //|doc = ${doc.replaceAll("\n", "\\$")},
```

Comment:
For testing; will remove change before merge.

Reply:
Understandable. I have a similar change on #3611 branch as well.

Reply:
Removed in 5da58dc09.